Source/WebGPU/WGSL/PhaseTimer.h

```diff
@@ -100,7 +100,7 @@ class PhaseTimer {
     }
 
 private:
-    const char* m_phaseName;
+    const char* m_phaseName { nullptr };
```

Comment:
I guess this is OK. If the Knoll phase name was ever appended to the vector, we’d have a problem later anyway. but this does no harm.